Orange/distance/__init__.py

```diff
@@ -194,26 +202,32 @@ def fit(self, data, axis=1):
             x = x.T
         n, m = x.shape
         if n <= m:
-            raise ValueError('Too few observations for the number of dimensions.')
+            raise ValueError('Too few observations for the number of '
+                             'dimensions.')
```

Comment:
Do it like this to avoid breaking the string:

```
            raise ValueError(
                'Too few observations for the number of dimensions.')
```